### DISCUSSION TITLE
single quote strings.

### DISCUSSION THREAD

**Discussion by flamypow** [jeffreylanters/react-unity-webgl#170] [Question and Answer] [answered]
what happens to the string exactly when I do unitycontext.send like so:

unityContext.send('GameController', 'GetString', 'a string in single quotes but also have "double" quotes in it'); if I put it in a gui.text = ($"{said string}") it actually prints out correctly but I do not know how to make said string into a workable format for visual studio.

**Comment by jeffreylanters** [comment, marked as answer]
Hi Flamypow, I'm not quite sure what you mean. Could you try to clearify your problem or question?

**Reply by flamypow** [reply]
you use unityContext.send passing in a string that is in single quotes, going from javascript into c sharp. c sharp use single quote as character only. But it kinda works for some reason, I don't know why it does and I don't know how to work with that string that is in single quote and have double quote in it.

**Reply by jeffreylanters** [reply]
If I'm understanding you correctly, when sending `'Hello "planet" earth'` to Unity, the double quotes will be converted into escaped characters like: `"Hello \"planet\" earth"`. This is natively supported by dot net, so when using that string in a GUI or UI element it will just render `Hello "planet" earth`.

**Reply by flamypow** [reply]
oh cool it escapes for you. Didn't know that, thanks
